scripts/run_all_tests.sh

```diff
@@ -6,7 +6,7 @@ PSQL=psql
 TMP_ROOT=/tmp/lanterndb
 TMP_OUTDIR=$TMP_ROOT/tmp_output
 FILTER="${FILTER:-}"
-DB_USER="${DB_USER:-}"
+DB_USER="${DB_USER:-postgres}"
```

Comment:
I think we can set default to os user 
```suggestion
DB_USER="${DB_USER:-$USER}"
```

also this part can be removed https://github.com/lanterndata/lanterndb/blob/main/test/test_runner.sh#L10-L21

Reply:
Good catch! fixed, and also made sure to use "postgres" in place of "root" when root is the user.
This is useful for when we are in docker containers